backend/src/lastwill/lastwill.service.ts

```diff
@@ -19,9 +29,13 @@ export class LastWillService {
     private readonly lastwillDbService: LastWillDBService,
   ) {}
 
-  // TODO: implement in other issue
   async getFullTextLastWill(id: string, userId: ObjectId) {
-    throw new NotImplementedException()
+    const lastWill = await this.lastwillDbService.findFullById(id, userId)
+
+    if (!lastWill) {
+      throw new NotFoundException()
+    }
```

Comment:
```suggestion
    if (!lastWill) throw new NotFoundException()
```
Guard clauses look better as one liners